addons/l10n_eg_edi_eta/views/account_move_view.xml

```diff
@@ -0,0 +1,51 @@
+<odoo>
+    <data>
+        <record id="action_sign_invoices" model="ir.actions.server">
+            <field name="name">Sign invoices</field>
+            <field name="type">ir.actions.server</field>
+            <field name="state">code</field>
+            <field name="model_id" ref="account.model_account_move"/>
+            <field name="binding_model_id" ref="account.model_account_move"/>
+            <field name="code">
+                action = records.action_post_sign_invoice()
+            </field>
+        </record>
+
+
+        <record id="view_move_form_inherit" model="ir.ui.view">
+            <field name="name">view_move_form_inherit</field>
+            <field name="model">account.move</field>
+            <field name="inherit_id" ref="account.view_move_form"/>
+            <field name="priority" eval="40"/>
+            <field name="arch" type="xml">
+                <xpath expr="//header/button[@name='action_post']" position="after">
+                    <button name="action_post_sign_invoice" type="object"
+                            class="oe_highlight"
+                            groups="account.group_account_invoice"
+                            string="Sign Invoice"
+                            attrs="{'invisible': ['|', '|', ('country_code', '!=', 'EG'), ('l10n_eg_is_signed', '=', True), ('state', '!=', 'posted')]}"/>
+                    <button name="action_get_eta_invoice_pdf" type="object"
```

Comment:
As discussed, better to move this button as class="oe_link" in the "ETA E-Invoice" page below as it is not something that is part of the main flow, but just something that customers might need to give more clarity to their customers.  (would not be bad to have some comment about it somewhere)